frontend/src/lib/components/HelpButton/HelpButton.tsx

```diff
@@ -139,14 +140,27 @@ export function HelpButton({
                             </LemonButton>
                         </a>
                     )}
+                    {!isPromptVisible && (
+                        <LemonButton
+                            icon={<IconTrendUp />}
+                            status="stealth"
+                            fullWidth
+                            onClick={() => {
+                                promptAction('activation-checklist')
+                                hideHelp()
+                            }}
+                        >
+                            How to be successful with PostHog
```

Comment:
On the project homepage, this button does the same thing as "Explain this page" 🤔 
<img width="308" alt="Screen Shot 2022-09-26 at 18 10 21" src="https://user-images.githubusercontent.com/4550621/192327591-920846ba-9091-4954-b477-968cf114fc65.png">